scripts/verify-codama-idls.sh

```
@@ -55,6 +55,16 @@ trap '
 	fi
 ' EXIT
 
+if [[ -z "${HOME:-}" ]]; then
+	export HOME="$ROOT/.cache/home"
+fi
+mkdir -p "$HOME"
+if [[ "$(uname)" == "Linux" ]] && command -v gcc >/dev/null 2>&1; then
+	export CC="$(command -v gcc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)